src/test/java/org/sonatype/cs/metrics/util/HelperServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package org.sonatype.cs.metrics.util;
```

Comment:
Beyond the scope of this PR, but there is room for improvement on the HelperService. Just adding this here for a reminder to myself to create a story/task/whatever we decide to do in Jira.